lib/tasks/serialisations.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+namespace :serialisations do
+  desc 'Rerun serialisations for all published content'
+  task rerun: :environment do
```

Comment:
Bit pedantic, but, could we rename this task to "contributions:serialise", because a serialisation is the output of the job, not the job itself, and this task is not checking for pre-existing serialisation jobs to rerun but just serialising all published contributions.